### PR TITLE
Add FAQ for when to expect release schedule info

### PR DESCRIPTION
**Pull request by priyanka-ravi**
resolves fluxcd/website#596

### CODE REVIEW

**Comment by dholbach** [comment]
Can you please sign your commits like so...?

In your local branch, run: git rebase HEAD~1 --signoff
Force push your changes to overwrite the branch: git push --force-with-lease origin release-schedule-faq

File: content/en/docs/faq.md [hunk 1]
```diff
@@ -385,3 +385,9 @@ to help you migrate to Flux v2:
 - [How to manage multi-tenant clusters with Flux v2](https://github.com/fluxcd/flux2-multi-tenancy)
 - [Migrate from Helm Operator to Flux v2](/docs/migration/helm-operator-migration/)
 - [How to structure your HelmReleases](https://github.com/fluxcd/flux2-kustomize-helm-example)
+
+## Release questions
+
+### Where can I find information on how long versions are supported and how often to expect releases?
+
+Releases are as needed for now and a release page with the schedule will be created when Flux reaches general availability status. We understand the importance of having a release schedule and are aware that such a schedule will support the users' ability to plan for an upgrade in advance.
```
Comment (dholbach) [inline content/en/docs/faq.md:393]:
```suggestion
Releases are as needed for now and a release page with the schedule will be created when Flux reaches general availability status. We understand the importance of having a release schedule and are aware that such a schedule will support the users' ability to plan for an upgrade in advance.

```

Comment (dholbach) [reply content/en/docs/faq.md:393]: Maybe point to `/roadmap` for now as well to at least point to some available information?

Maybe also point out that `flux2` in 2021 had at least monthly releases, often more and all controllers together 100+ releases in that time-frame? That at least would demonstrate some of the release velocity and while not a strict guarantee - at least an indication of progress (even if in a "pre-release" kind of fashion still).

Comment (stefanprodan) [reply content/en/docs/faq.md:393]: We can say that the release cadence is at least once per month.

Comment (priyanka-ravi) [reply content/en/docs/faq.md]: Let me know if the update looks good! If y'all think I should add in more info about our current releases let me know! Thanks!!

**Review by scottrigby — APPROVED** [review]
👍

**Comment by scottrigby** [comment]
The linkchecker was failing on an unrelated file, so I merged this. We should look into that. I tried running it twice. Example failing run: https://github.com/fluxcd/website/runs/4972626006?check_suite_focus=true#step:9:56

**Pull request merged**